core/tests/protractor_utils/action.js

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Utilities for performing actions on element.
+ */
+
+var waitFor = require('./waitFor.js');
+
+var clear = async function(inputName, inputElement) {
+  await click(inputName, inputElement);
```

Comment:
Why don't we check for clickability here?

Reply:
I was not sure whether the input element will pass isClickable check (I was considering buttons and links to only pass isClickable check!)

I'll verify and update!